static/js/public/about/side-navigation.js

```diff
@@ -0,0 +1,60 @@
+class SideNavigation {
```

Comment:
Is JS really necessary for the side nav?

Reply:
yes, we need it for mobile to hide/show the navigation